presentation/src/main/res/values/strings.xml

```diff
@@ -187,7 +186,7 @@
     <string name="backup_selected_backup_error_message">Couldn\'t read backup file</string>
     <string name="backup_selected_backup_details_title">Backup details</string>
     <string name="backup_restore_dialog_title">Backups</string>
-    <string name="backup_restore_dialog_summary" translatable="false">/QUIK/Backups</string>
+    <string name="backup_restore_dialog_summary" translatable="false">/UP-Messages/Backups</string>
```

Comment:
This is not a string; this is a configuration. Are you sure it's ok to change this?

Reply:
It's the default directory it creates for backups so yes it's ok to change and even necessary to change since it's user facing